lib/templates/imageFunction.js

```diff
@@ -0,0 +1,26 @@
+const jimp = require("jimp");
+
+// Function used to mimic next/image and sharp
+exports.handler = async (event) => {
+  const { url, w = 500, q = 75 } = event.queryStringParameters;
+  const width = parseInt(w);
+  const quality = parseInt(q);
+
+  const imageUrl = url.startsWith("/")
+    ? `${process.env.URL || "http://localhost:8888"}${url}`
```

Comment:
not sure about this ||, but i think it's fine since ntl dev uses 8888 as its default port right?

Reply:
this is probably okay in the short term, but the port is configurable. quick fix is the create a `.env` with `URL=http://localhost:5432` or whatever to override locally